modules/lightning_features/lightning_media/src/BundleResolverInterface.php

```diff
@@ -23,4 +23,12 @@
    */
   public function getBundle($input);
 
+  /**
+   * Returns all possible bundles for the field type(s) this plugin supports.
+   *
+   * @return MediaBundleInterface[]
+   *   Applicable media bundles, keyed by ID.
+   */
+  public function getPossibleBundles();
```

Comment:
`BundleResolverBase` already has an implementation of this which I mostly kept untouched. Instead of returning all possible bundles it limits the bundles to those that have a matching source field definition.